Formula/miller.rb

```diff
@@ -17,17 +17,13 @@ class Miller < Formula
 
   depends_on "autoconf" => :build
   depends_on "automake" => :build
+  depends_on "go" => :build
   depends_on "libtool" => :build
 
   uses_from_macos "flex" => :build
```

Comment:
i don't think we need these anymore right?
```suggestion
  depends_on "go" => :build
```

Reply:
@branchvincent correct ... thank you!

Reply:
Whoops, forgot about that. :grin: